### PR TITLE
feat: Payment Review Page

### PR DESCRIPTION
**Pull request by rhysbarrett**
### JIRA link

https://companieshouse.atlassian.net/browse/SR-64

### Change description

This commits implements and tests the Payment Review page.

### Work Checklist

- [x] Tests added where applicable
- [x] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

---

### Merge Instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/modules/config-handler/ConfigValidation.schema.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ export const configValidationSchema = Joi.object({
   COOKIE_DOMAIN: Joi.string().required(),
   COOKIE_SECRET: Joi.string().required(),
   CACHE_SERVER: Joi.string().required(),
-  COOKIE_EXPIRATION_IN_SECONDS: Joi.string().required()
+  COOKIE_EXPIRATION_IN_SECONDS: Joi.string().required(),
+  DOCUMENT_AMENDMENT_FEE: Joi.number()
```
Comment (akaJSaunders) [inline src/modules/config-handler/ConfigValidation.schema.ts:11]: Should this not be required?

File: src/routes/routes.ts [hunk 2]
```diff
@@ -30,3 +38,8 @@ routes.post(ADDRESS_TO_REMOVE_PAGE_URI, addressToRemoveController.processForm);
 
 routes.get(DOCUMENT_DETAILS_PAGE_URI, documentDetailsController.renderView);
 routes.post(DOCUMENT_DETAILS_PAGE_URI, documentDetailsController.processForm);
+
+routes.get(PAYMENT_REVIEW_PAGE_URI, paymentReviewController.renderView);
+routes.post(PAYMENT_REVIEW_PAGE_URI, paymentReviewController.continue);
+
+
```
Comment (akaJSaunders) [inline src/routes/routes.ts:45]:
```suggestion
```

File: src/modules/config-handler/ConfigValidation.schema.ts (hunk 1 above)
Comment (rhysbarrett) [reply src/modules/config-handler/ConfigValidation.schema.ts:11]: Ah, yes, good spot. I think I thought required was implicit.

File: src/routes/routes.ts (hunk 2 above)
Comment (rhysbarrett) [reply src/routes/routes.ts:45]: Gosh darn it, I thought I'd configured my IDE to trim trailing whitespace, why does this keep happening. Thanks!

**Review by akaJSaunders — APPROVED** [review]

**Pull request merged**